src/api/cvc4cpp.h

```diff
@@ -1954,6 +1954,11 @@ class CVC4_PUBLIC Grammar
    */
   void addRules(Term ntSymbol, std::vector<Term> rules);
 
+  /**
+   * Nullary constructor. Needed for the python api.
```

Comment:
Super minor, but lets call this the Cython API for clarity. Technically the old SWIG python bindings haven't been deprecated yet.

Reply:
The old SWIG bindings are actually already removed on `master` :) I don't have a preference w.r.t. Cython vs Python API though.

Reply:
Oh great, didn't realize that! Either is fine then.